packages/create-sitecore-jss/src/templates/vue/data/routes/da-DK.yml

```diff
@@ -0,0 +1,4 @@
+# in order to demonstrate translation on the Styleguide page, we need to have a stub on the homepage for that language.
```

Comment:
see other initializers for the language ymls. We'll need to replace da-DK in the filename with our token replacer {{language}} for example.